api_collections/config_file.py

```diff
@@ -0,0 +1,5 @@
+BASE_URL = "https://restful-booker.herokuapp.com/"
+API_KEY = "dfkjghsdkjfhgkjshdfjk"
+USERNAME = "admin"
+PASSWORD = "password123"
+TOKEN = "85896a61e7771c9"
```

Comment:
you should use .json
I use .py at the lesson because I want to show more and not to spend time fot implementing fixture at the lesson